apps/judicial-system/api/src/app/modules/case-list/dto/caseStatistics.input.ts

```diff
@@ -19,3 +19,60 @@ export class CaseStatisticsInput {
   @Field(() => String, { nullable: true })
   readonly institutionId?: string
 }
+
+@InputType()
```

Comment:
decided to keep this all within the same file since they are all pretty small, an maybe we can just combine them and have one shared input between types. Maybe I will change that in a follow-up work.

Reply:
I wonder if statistics shouldn't be in a separate module?

Reply:
Agree, already implemented [here](https://app.asana.com/1/203394141643832/project/1199153462262248/task/1210998427054268)